test.py

```diff
@@ -0,0 +1,247 @@
+import argparse
+import json
+import os
+import pathlib
+import platform
+import shutil
+import subprocess
+import sys
+import urllib.parse
+import urllib.request
+import zipfile
+
+def DownloadAndUnzip (url, dest, platform):
+    # https://github.com/GRAPHISOFT/archicad-api-devkit/releases/download/<tag>/ <filename>
+    # https://github.com/GRAPHISOFT/archicad-api-devkit/releases/latest/download/ <filename>   - not used, as there isn't a single latest version
+
+    version = url.split('/')[-1]
+    fileName = f'API.Development.Kit.{platform}.{version}.zip'

```

Comment:
It would be better if the whole script used f-strings to concatenate instead of the '+'. Not too important, but if you have the time, can you replace them?

Reply:
Fixed